Runtimes/Supplemental/Observation/CMakeLists.txt

```diff
@@ -0,0 +1,110 @@
+cmake_minimum_required(VERSION 3.29)
+# TODO before requiring CMake 4.1 or later
+# and/or enforcing CMP0195, please check/update
+# the implementation  of `emit_swift_interface`
+# in `EmitSwiftInterface.cmake`
+# to ensure it keeps laying down nested swiftmodule folders
+
+if(POLICY CMP0157 AND CMAKE_Swift_COMPILER_USE_OLD_DRIVER)
+  cmake_policy(SET CMP0157 OLD)
+endif()
+
+if($ENV{BUILD_NUMBER})
+  math(EXPR BUILD_NUMBER "$ENV{BUILD_NUMBER} % 65535")
+  set(BUILD_NUMBER ".${BUILD_NUMBER}")
+endif()
+project(SwiftObservation
+  LANGUAGES Swift CXX
+  VERSION 6.1.0${BUILD_NUMBER})
+
+if(NOT PROJECT_IS_TOP_LEVEL)
+  message(SEND_ERROR "Swift Synchronization must build as a standalone project")
```

Comment:
```suggestion
  message(SEND_ERROR "Swift Observation must build as a standalone project")
```

Reply:
oof ... thanks! Is there a previous thread about the `plist.in` copying? The thread on this PR is related to `CMakeConfig.h` generated in the Swift build.